en.json

```
@@ -1366,8 +1369,10 @@
         "Tutorial.Intro.BadgesDisabilitySpeech" : "Speech Impaired",
         "Tutorial.Intro.BadgesTitle" : "Badges",
         "Tutorial.Intro.BadgesDescription" : "Badges are small icons which show aspects of a user. Here are a few important badges to remember. The disability badges can be self assigned though the NeosBot.",
-        "Tutorial.Intro.HelpSite" : "Help",
-       
+        "Tutorial.Intro.HelpSite" : "Help
```

```suggestion
        "Tutorial.Intro.HelpSite" : "Help",
```
You lost a quote and a comma there